src/containers/Review/AssignmentSectionRow.tsx

```diff
@@ -27,7 +28,7 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = (props) => {
     if (!assignment) return
 
     try {
-      await assignSectionToUser({ assignment, sectionCode })
+      await assignSectionToUser({ assignment, assignerId: userId, sectionCode })
```

Comment:
what do you think about getting userId in the hook vs passing it through as props ?

Reply:
Hmmm for this one it should be fine. That was my first trial, but I was having a hook error thrown because the other place calling wasn't a React component. But I can change for this one

Reply:
Done: 21d664a

Reply:
i was meaning something like this: `https://github.com/openmsupply/application-manager-web-app/compare/assignerId`